Project.toml

```diff
@@ -29,8 +29,8 @@ Pidfile = "1.3"
 Pkg = "1.6"
 Random = "1.6"
 RandomExtensions = "0.4.2"
-Singular_jll = "~403.209.900"
+Singular_jll = "~403.210.1000"
 Statistics = "1.6"
 julia = "1.6"
 lib4ti2_jll = "1.6.10"
-libsingular_julia_jll = "~0.40.5"
+libsingular_julia_jll = "~0.40.6"
```

Comment:
Maybe we should first apply this in a separate PR, merge that and make a release?

Reply:
I would support this separation.

Reply:
done with https://github.com/oscar-system/Singular.jl/pull/723